src/automata/include/automata/ata.hpp

```diff
@@ -170,8 +190,16 @@ AlternatingTimedAutomaton<LocationT, SymbolT>::make_symbol_step(
 		models.push_back(new_states);
 	}
 	// We were not able to make any transition.
-	if (models.empty()) {
-		return {};
+	if (models.empty() || std::any_of(std::begin(models), std::end(models), [](const auto &model) {
+		    return model.empty();
+	    })) {
+		// We have a sink location, the next configuration is the singleton {(sink, 0)}.
+		if (sink_location_.has_value()) {
+			return {{State<LocationT>{sink_location_.value(), 0}}};
```

Comment:
Ah - I was wondering how we handle the sink - this makes more sense, as it avoids adding extra transitions.